### PR TITLE
chore: multidevices tests

### PR DESCRIPTION
**Pull request by D4ryl00**
Create tests for multidevices:
- TestMultiDevices_Basic

### CODE REVIEW

**Review by moul — APPROVED** [review]
LGTM, can you rebase your commits?

**Comment by D4ryl00** [comment]
> LGTM, can you rebase your commits?

done

File: go/pkg/bertyprotocol/store_metadata_test.go [hunk 1]
```diff
@@ -656,3 +656,141 @@ func TestMetadataGroupsLifecycle(t *testing.T) {
...
+	testutil.FilterStabilityAndSpeed(t, testutil.Unstable, testutil.Slow)
+
+	memberCount := 2
+	deviceCount := 3
+	totalDevices := memberCount * deviceCount
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*30)
+	defer cancel()
+
+	peers, _, cleanup := createPeersWithGroup(ctx, t, "/tmp/multidevices_test", memberCount, deviceCount)
```
Comment (Jorropo) [inline go/pkg/bertyprotocol/store_metadata_test.go:670]:
```suggestion
	tDir, err := ioutil.TempFile("/tmp", "multidevices_test")
	defer os.Remove(tDir.Name())
	require.NoError(t, err)
	peers, _, cleanup := createPeersWithGroup(ctx, t, tDir.Name(), memberCount, deviceCount)
```

**Review by Jorropo — CHANGES_REQUESTED** [review]

File: go/pkg/bertyprotocol/store_metadata_test.go (hunk 1 above)
Comment (D4ryl00) [reply go/pkg/bertyprotocol/store_metadata_test.go:670]: I did as other tests in this file, but `createPeersWithGroup` doesn't use the path currently. I don't know why it's not used, but if we must create temp file, I think it should be done in the helper function `createPeersWithGroup`, not in the test directly. But it's an other problem ;)
So I think that I did is good to be homogeneous with other tests.

Comment (Jorropo) [reply go/pkg/bertyprotocol/store_metadata_test.go:670]: > 
> 
> I did as other tests in this file, but `createPeersWithGroup` doesn't use the path currently. I don't know why it's not used, but if we must create temp file, I think it should be done in the helper function `createPeersWithGroup`, not in the test directly. But it's an other problem ;)
> So I think that I did is good to be homogeneous with other tests.

As you wish, I just don't really like static path (I guess this is still mergeable BTW)

**Review by moul — APPROVED** [review]
LGTM

**Pull request merged**